server/middleware/apiProxy.js

```diff
@@ -21,7 +21,8 @@ function apiProxy(dataAdapter) {
     api.path = apiProxy.getApiPath(req.path);
     api.api = apiProxy.getApiName(req.path);
     api.headers = {
-      'x-forwarded-for': apiProxy.getXForwardedForHeader(req.headers, req.ip)
+      'x-forwarded-for': apiProxy.getXForwardedForHeader(req.headers, req.ip),
+      'x-http-method-override': req.headers['x-http-method-override']
```

Comment:
You don't want to check  if `req.headers['x-http-method-override']` field is set ?